### PR TITLE
table: export `WritableCols`

### PR DESCRIPTION
**Pull request by coocood**
`WritableCols` will be used in ddl_test and in executor for update multiple table.

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala PTAL

File: table/table.go [hunk 1]
```diff
@@ -72,6 +72,9 @@ type Table interface {
 	// Cols returns the columns of the table which is used in select.
 	Cols() []*Column
 
+	// WritableCols returns the writable columns of the table.
```
Comment (shenli) [inline table/table.go]: The comment should be more detailed.

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**